docs/setup-cluster/deploy-cluster/slurm/upgrade-on-hpc.rst

```diff
@@ -0,0 +1,39 @@
+.. _upgrade-on-hpc:
+
+#################################
+ Upgrade Determined on Slurm/PBS
+#################################
+
+This procedure describes how to deploy an upgraded Determined on an HPC cluster managed by the Slurm
```

Comment:
`deploy an upgraded Determined` doesn't sound right.  As a suggestion:

This procedure describes how to upgrade Determined on an HPC cluster managed ...